overlay/make-package-set/full.nix

```diff
@@ -17,6 +17,8 @@
   fetchCrateAlternativeRegistry ? _: throw "fetchCrateAlternativeRegistry is required, but not specified in makePackageSet",
   release ? null,
   rootFeatures ? null,
+  hostPlatformCpu ? null,
```

Comment:
We might need to provide a different set of features and CPU for the build platform in `make-package-set/simplified.nix`, but I doubt that there's any crate out there makes use of it, so I guess this should be fine.

Reply:
Would you mind opening a GitHub issue for this, @trha? We should revisit it for completeness' sake.